src/components/NavBar/NavigationBar.tsx

```diff
@@ -43,45 +59,52 @@ function NavigationBar(): JSX.Element {
     };
   }, []);
 
-  // Toggles the navigation
-  function toggleNav(): void {
-    setIsNavVisible(!isNavVisible);
-  }
-
-  /** Toggle the language of the website between English and Spanish. Close menu bar after toggling if on mobile. */
-  function toggleLanguage(): void {
-    i18n.changeLanguage(i18n.language === 'en' ? 'es' : 'en');
-    if (isMobile) {
-      toggleNav();
+  /**
+   * Sets isScrolled depending on how far the user has scrolled
+   */
+  function handleScroll(): void {
+    const scrollThreshold = 20;
+    const offset = window.scrollY;
+    if (offset > scrollThreshold) {
+      setIsScrolled(true);
+    } else {
+      setIsScrolled(false);
     }
   }
 
-  const {t, i18n} = useTranslation('menu');
+  /**
+   * Event listener for scrolling
+   */
+  useEffect(() => {
+    window.addEventListener('scroll', handleScroll);
+    return function (): void {
+      window.removeEventListener('scroll', handleScroll);
+    };
+  });
```

Comment:
Does this need a dependency array? (Not sure if it does, just asking)

Reply:
Seemed to be working without it, but I'm not sure why so I just added it to be safe.